tests/group_functions/group_scan.h

```diff
@@ -436,7 +436,7 @@ void check_scan_over_group(sycl::queue& queue, sycl::range<D> range, OpT op,
                 // Use the local id of the item in the sub-group to place
                 // results of the scan operation in the order of the items.
                 auto sg_index = sub_group.get_group_linear_id() *
-                                    sub_group.get_local_linear_range() +
+                                    sub_group.get_max_local_range()[0] +
```

Comment:
```suggestion
                                    sub_group.get_max_local_range() +
```
I don't believe we need the indexing. Just a nit though.